src/picknik_ur_studio_integration_testing/test/launch/test_basic_ur5e_objectives.test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 PickNik Inc.
+# All rights reserved.
+#
+# Unauthorized copying of this code base via any medium is strictly prohibited.
+# Proprietary and confidential.
+
+
+import os
+import sys
+import unittest
+
+import launch_testing
+
+from picknik_ur_studio_integration_testing.generate_agent_plus_drivers_launch_description import (
+    generate_agent_plus_drivers_launch_description,
+)
+
+
+def generate_test_description():
+    return generate_agent_plus_drivers_launch_description(
+        gtest_name="test_basic_ur5e_objectives",
+        env_vars={
+            # Load site config for simulated UR-5e
+            "STUDIO_CONFIG_PACKAGE": "picknik_001_ur5e_config",
```

Comment:
Replace this with one of the config packages in this repo, e.g., `picknik_ur5e_site_config` or `picknik_ur5e_mock_hw_config`.

This right now only works because we still have the `picknik_ur5e_configs` repo in our submodule abyss